variables.tf

```diff
@@ -1,3 +1,8 @@
+variable "create_alb" {
```

Comment:
Thinking about this a little harder, it's more than just creation or non-creation of the alb, it's everything. Let's have the variable reflect that. `create_resources` seems a little more suitable.